### PR TITLE
fix function names in elixir docs and remove deprecated additional info

### PR DESCRIPTION
**Pull request by vivianyentran**
## DESCRIBE YOUR PR
- Fix for #3213 
- Fix references to context function name
- Remove deprecated additional info section

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Comment by vivianyentran** [comment]
@sl0thentr0py Can you help review this?

File: docs/platforms/elixir/enriching-events/context/index.mdx [hunk 1]
```diff
@@ -17,7 +17,7 @@ If you need to be able to search on custom data, [use tags](../tags) instead.
 
 The best way to attach custom data is with a structured context. A context must always be an object and its values can be arbitrary.
```
Comment (lizokm) [inline docs/platforms/elixir/enriching-events/context/index.mdx:18]:
```suggestion
The best way to attach custom data is with a structured context. A context must always be an object while its values can be arbitrary.
```

**Comment by lizokm** [comment]
@sl0thentr0py gentle nudge to take a look at this when you have a moment please :)

**Review by sl0thentr0py — APPROVED** [review]
yep, sorry was out on vacation!

**Comment by lizokm** [comment]
Thanks so much @sl0thentr0py

**Pull request merged**